tfe/resource_tfe_workspace.go

```diff
@@ -767,3 +806,12 @@ func resourceTFEWorkspaceImporter(ctx context.Context, d *schema.ResourceData, m
 func warnWorkspaceTagsCasing(ctx context.Context, tag string) {
 	log.Printf("[WARN] The tag \"%s\" contains uppercase characters that will be transformed by the API. Please update your configuration to lowercase tags in order to avoid conflicts with state.", tag)
 }
+
+func errWorkspaceSafeDeleteWithPermission(workspaceID string, err error) error {
+	if err != nil {
+		if strings.HasPrefix(err.Error(), "conflict") {
+			return fmt.Errorf("Error deleting workspace %s: %w\nTo delete this workspace without destroying the managed resources, add force_delete=true to the resource config.", workspaceID, err)
```

Comment:
```suggestion
			return fmt.Errorf("Error deleting workspace %s: %w\nTo delete this workspace without destroying the managed resources, add force_delete = true to the resource config.", workspaceID, err)
```

Might as well show our best guess at a proper `terraform fmt` property 😬

Reply:
yes ! a great idea